encrypted-media/Google/encrypted-media-playback-encrypted-and-clear-sources.html

```diff
@@ -94,7 +94,7 @@
 
                 function startPlayback() {
                     if (playbackCount % 2) {
-                      video.src = '../content/test-vp8-vorbis-webvtt.webm';
+                      video.src = 'content/test-vp8-vorbis-webvtt.webm';
```

Comment:
This is just a clear file:
* We can copy this file from https://github.com/w3c/web-platform-tests/tree/master/media-source/webm.
* Since we don't actually need the tracks, copying and using test.webm from that same location is probably better.